### ISSUE TITLE
How do you use it?

### ISSUE THREAD

**Issue by sarg** [mkcms/gpx-mode#2]
Hi there, I've found your package through Sascha Chua's newsletter and gave it a try. It works fine for tracks created by a Samsung smartwatch and it'd be cool if the heartrate graph could be displayed when present. Another minor improvement could be that "Show elevation profile" would work as "Toggle elevation profile".

I've also checked if there is a way to display the map directly in Emacs and found this cool OSM viewer: https://github.com/minad/osm

It works great, though doesn't support displaying individual tracks.
```lisp
(defun gpx-show-map-osm (file track segment)
    (osm-gpx-show file))
```

Finally the main reason I'm writing this ticket - do you have a write up on how do you use the package?

**Comment by mkcms** [comment]
Hi.

> it'd be cool if the heartrate graph could be displayed when present.

I've considered it, but haven't got to actually trying to do it. More generally, we should be able to display any available chart like elevation profile.

> Another minor improvement could be that "Show elevation profile" would work as "Toggle elevation profile".

Nice idea, I will do it soon.

> I've also checked if there is a way to display the map directly in Emacs and found this cool OSM viewer: https://github.com/minad/osm

That's looks cool, thanks for sharing. I will add this information to the README.

> how do you use the package?

I wrote this package because I had hundreds of GPX files on my watch and I was cleaning them up. They had cryptic names, and previewing them online was very time-consuming, so here it is :) It's much quicker to just open the entire directory in Dired and filter them out there, with preview.

**Comment by mkcms** [comment]
The toggle button request was implemented in 9dabb3db976e5e87f7d1151447b905cbe4a376ef.
